Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js

```diff
@@ -18,6 +18,7 @@
     function syntaxHighlight(json) {
         json = JSON.stringify(json, undefined, 4);
         json = json.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;');
+        json = json.replace(/ /g, '&nbsp;');
```

Comment:
Shouldn't this just be a html-text-encode call? There are likely other characters than '>', '<', ' ' that would need preserving.